npm-shrinkwrap.json

```diff
@@ -2555,23 +2555,10 @@
       "from": "form-data@>=2.1.1 <2.2.0",
       "resolved": "https://registry.npmjs.org/form-data/-/form-data-2.1.2.tgz",
       "dependencies": {
-        "async": {
-          "version": "0.9.2",
-          "from": "async@>=0.9.0 <0.10.0",
-          "resolved": "https://registry.npmjs.org/async/-/async-0.9.2.tgz",
-          "dev": true
-        },
-        "mime-db": {
```

Comment:
Actually, looks like you were right about this one. `mime-db` is a dependency of `mime-types`, and its the reason why https://github.com/resin-io/etcher/pull/1379 is failing. I'll bring it back!